test/tests.py

```diff
@@ -433,6 +433,18 @@ def get_block_height(ip, port):
     if enabled("btcpayserver"):
         machine.wait_until_succeeds(log_has_string("nbxplorer", f"At height: {num_blocks}"))
 
+@test("trustedcoin")
+def _():
+    machine.wait_for_unit("bitcoind")
+    machine.wait_for_unit("clightning")
+
+    # Let's check the trustedcoin plugin was correctly initialized
+    machine.wait_until_succeeds(log_has_string("clightning", "plugin-trustedcoin[^^]\[0m\s+initialized plugin"))
+    machine.wait_until_succeeds(log_has_string("clightning", "plugin-trustedcoin[^^]\[0m\s+bitcoind RPC working"))
+    machine.wait_until_succeeds(log_has_string("clightning", "plugin-trustedcoin[^^]\[0m\s+tip: 0"))
+    machine.wait_until_succeeds(log_has_string("clightning", "plugin-trustedcoin[^^]\[0m\s+estimatefees error: none of the esploras returned usable responses"))
```

Comment:
Couldn't this test break if esplora gets fixed and suddenly returns usable responses?

Reply:
Ah, I don't understand the testing framework enough apparently :see_no_evil: just assumed testing for something being broken was odd.

Reply:
> so the trustedcoin should never get a response from the esploras.

Yes but the trustedcoin plugin could change it's output in that case. But I think the test is ok.